api/pkg/apis/v1alpha1/providers/target/helm/helm.go

```diff
@@ -741,7 +770,7 @@ func configureUpgradeClient(ctx context.Context, componentProps *HelmChartProper
 		upgradeClient.Namespace = deployment.Instance.Spec.Scope
 	}
 	upgradeClient.ResetValues = true
-	upgradeClient.Install = true
+	upgradeClient.Install = false
```

Comment:
Why change to false?

Reply:
When release not found, it will not install chart.

Reply:
Ok. If no side effect, it's safe to go with `false`.

Reply:
`upgrade.Install = true` should be used to implement the behavior like `helm upgrade --install`. Seems we can use helm upgrade client to cover both install and update case. @yanjiaxin534 , do you think we can do this to remove installClient?  And when upgrade client with install option encounter any issues, we should stop and don't need to try again with install client and omit the upgrade failure.